dll/shellext/ntobjshex/ntobjfolder.cpp

```diff
@@ -484,6 +484,12 @@ HRESULT CNtObjectFolder::GetInfoFromPidl(LPCITEMIDLIST pcidl, const NtPidlEntry
 {
     NtPidlEntry * entry = (NtPidlEntry*) &(pcidl->mkid);
 
```

Comment:
I would strip that empty line before your "if" in both files ntobjfolder.cpp and regfolder.cpp.
Aside from that LGTM.

Reply:
was addressed.